### PR TITLE
Key Authentication plugin updates for CE 2.3 (MERGE FOR 2.3 CE RELEASE ON 1/20)

### PR DESCRIPTION
**Pull request by tlblessing**
https://konghq.atlassian.net/browse/DOCS-1500

**Direct review links**

Aapo please review:
https://deploy-preview-2530--kongdocs.netlify.app/hub/kong-inc/key-auth/
https://deploy-preview-2530--kongdocs.netlify.app/hub/kong-inc/key-auth/#about-api-key-locations-in-a-request

Nate please review:
https://deploy-preview-2530--kongdocs.netlify.app/enterprise/2.2.x/developer-portal/administration/application-registration/enable-key-auth-plugin/#key-auth-params
https://deploy-preview-2530--kongdocs.netlify.app/enterprise/2.2.x/developer-portal/administration/application-registration/enable-key-auth-plugin/#about-api-key-locations-in-a-request

Per Nate (and Jordan), the new fields show up in Kong Mgr GUI since it reads the most current 2.3 key-auth schema  and generates the form based off of that.

### CODE REVIEW

**Review by nateslo — APPROVED** [review]
https://deploy-preview-2530--kongdocs.netlify.app/enterprise/2.2.x/developer-portal/administration/application-registration/enable-key-auth-plugin/#key-auth-params
https://deploy-preview-2530--kongdocs.netlify.app/enterprise/2.2.x/developer-portal/administration/application-registration/enable-key-auth-plugin/#about-api-key-locations-in-a-request

These look good to me 👍

**Review by bungle — APPROVED** [review]

File: app/_includes/md/plugins-hub/api-key-locations.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!---will be shared in future with key-auth-enc plugin, also shared with on-prem app reg key-auth --->
+
+Use cases for key locations:
+
+* Recommended: Use `key_in_header` (enabled by default) as the most common and
+  secure way to do service-to-service calls.
+* If you need to share links to browser clients, use `key_in_query` (enabled by default).
+  Note that query parameter requests can appear within application logs and URL browser bars,
+  which expose the API key.
+* If you are sending a form with a browser, such as a login form, use the `key_in_body`. This option is
```
Comment (lena-larionova) [inline app/_includes/md/plugins-hub/api-key-locations.md:10]:
```suggestion
* If you are sending a form with a browser, such as a login form, use `key_in_body`. This option is
```

**Review by lena-larionova — APPROVED** [review]
One tiny fix, otherwise LGTM.

**Comment by tlblessing** [comment]
Resolved merge conflict using GUI, will merge PR after successful build

**Pull request merged**